pkg/fftypes/jsonany.go

```diff
@@ -135,11 +136,18 @@ func (h *JSONAny) JSONObjectNowarn() JSONObject {
 	return jo
 }
 
+// Value ensures we write null to the DB for null values
+func (h *JSONAny) Value() (driver.Value, error) {
+	if h.IsNil() {
+		return nil, nil
```

Comment:
Previously when we stored `null` data to the DB, we were storing the string `"null"`. That's a pain as the query REST filtering doesn't let you check for that using our standard null-checking.

I hit confusion in the code too about storing `nil` and getting back `"null"` for other paths.

I did a good hunt around for any implications of this change on other stored objects, and didn't find any. We were using `JSONObject` in a lot more cases, which already stores `null` as `nil` (not `"null"`)